tests/testthat/test-previous-issues.R

```diff
@@ -0,0 +1,176 @@
+test_that("Error message for invalid .bib files to read.bib #3", {
+  bib <- system.file("bib/badFormat.bib", package = "bibtex")
+
+  message <- expect_message(read.bib(bib))
+
+  expect_snapshot_output(message$message)
+})
+
+
+test_that("arrange.authors when family names have white space #6", {
+  tmp <- tempfile(fileext = ".bib")
+  entry <- "@Article{newspaper,
```

Comment:
Should these be written in a stand alone file within `inst/bib/*` instead of directly inline with the test?

Reply:
Good idea, already done except for issue #35 , that specifically ask for reading from a string, not a file